pkg/accounts/adaptor/controller/account.ts

```diff
@@ -422,27 +441,63 @@ export class AccountController {
       return followings;
     }
 
-    const accounts = await this.fetchService.fetchManyAccountsByID(
+    const accountsRes = await this.fetchService.fetchManyAccountsByID(
       Result.unwrap(followings),
     );
-
-    if (Result.isErr(accounts)) {
-      return accounts;
+    if (Result.isErr(accountsRes)) {
+      return accountsRes;
+    }
+    const accounts = Result.unwrap(accountsRes);
+
+    // Note: remove duplicated accounts
+    const accountIDs = [...new Set(accounts.map((v) => v.getID()))];
```

Comment:
```suggestion
    const accountIDs = new Set(accounts.map((v) => v.getID()));
```
重複を取り除いて列挙するのであれば, `Array` へスプレッド展開する必要は無いと思います. 531 行目も同様です.